variables.tf

```diff
@@ -1,21 +1,75 @@
-variable "lambda_name" {}
+variable "enabled" {
+  type        = "string"
+  default     = "true"
+  description = "Toggle the creation and destruction of all resources in this module"
+}
 
-variable "runtime" {}
+locals {
+  // transform the public `enabled` variable into a count integer
+  enabled = "${lower(var.enabled) == "true" ? 1 : 0}"
+}
 
-variable "lambda_zipfile" {}
+variable "lambda_name" {
+  type        = "string"
```

Comment:
`type = string` is not needed, because it is by default.

Reply:
Yeah, but that is bad practice. Types should be declared explicitly.